.github/workflows/tests.yaml

```diff
@@ -22,12 +22,12 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -e .
-          pip install pytest pytest-cov
+          pip install pytest pytest-cov pytest-mpl
       - name: Test with pytest
         run: |
-          pytest --cov=topostats
+          pytest --cov=topostats --mpl
```

Comment:
It's a useful plugin but its not very obvious how it works, you have to `return fig` at the end, then the first time you run with the option `pytest --mpl-generate-images=<path/to/save/images/to>` and it will `skip` tests with the `@pytest.mark.mpl_image_compare` fixture. When you then run the tests again without the option it compares the images generate by the test to those that existed.

Fairly circular chicken and egg situation, but its been useful as I move to refactoring to classes to use the images I generated with these functions to test I get the correct images with the classes.